Makefile

```diff
@@ -5,9 +5,10 @@ build:
 
 test:
 	docker run --rm -v "${PWD}/test":/tmp aadt my_address_file_geocoded.csv
+	docker run --rm -v "${PWD}/test":/tmp aadt my_address_file_geocoded.csv 800
```

Comment:
add this to the github actions test run as well.

(I guess at some point, we might be able to just tell github actions to `make test`?  I always forget to change after I change in the Makefile.)

Reply:
thanks for catching that